PyInstaller/utils/git.py

```diff
@@ -1,59 +0,0 @@
-#-----------------------------------------------------------------------------
-# Copyright (c) 2005-2023, PyInstaller Development Team.
-#
-# Distributed under the terms of the GNU General Public License (version 2
-# or later) with exception for distributing the bootloader.
-#
-# The full license is in the file COPYING.txt, distributed with this software.
-#
-# SPDX-License-Identifier: (GPL-2.0-or-later WITH Bootloader-exception)
-#-----------------------------------------------------------------------------
-"""
-This module contains various helper functions for git DVCS.
-"""
-
-import os
-
-from PyInstaller.compat import exec_command, exec_command_rc
-
-try:
-    WindowsError
-except NameError:
-    # Not running on Windows
-    WindowsError = FileNotFoundError
-
-
-def get_repo_revision():
-    path = os.path  # shortcut
-    gitdir = path.normpath(path.join(path.dirname(os.path.abspath(__file__)), '..', '..', '.git'))
-    cwd = os.path.dirname(gitdir)
-    if not path.exists(gitdir):
-        try:
-            from PyInstaller.utils._gitrevision import rev
```

Comment:
I think this was the only use of `PyInstaller.utils._gitrevision` (which supplies git commit/revision information for archive snapshots generated via git-archive), so we can remove that as well. (Plus the corresponding [entry in .gitattributes](https://github.com/pyinstaller/pyinstaller/blob/5bd9dd70b4c792103d048ecf43933ba8d71e0993/.gitattributes#L25))